pkg/cmd/oauth-apiserver/cmd.go

```diff
@@ -147,16 +154,13 @@ func (o *OAuthAPIServerOptions) NewOAuthAPIServerConfig() (*apiserver.Config, er
 		&apiserverstorage.ResourceConfig{},
 		specialDefaultResourcePrefixes,
 	)
-	if len(o.RecommendedOptions.Etcd.EncryptionProviderConfigFilepath) != 0 {
```

Comment:
do we have e2e tests demonstrating this still works?

Reply:
[encryption tests](https://github.com/openshift/cluster-authentication-operator/blob/master/test/e2e-encryption/encryption_test.go) have been run on this PR (cluster from cluster bot) and they have passed successfully